utils/validations.py

```diff
@@ -0,0 +1,34 @@
+import re
+from django.core.exceptions import ValidationError
```

Comment:
Now, because we are also using GraphQL, let that layer handle the exceptions/ errors that raise before they are passed down to django. 
you can import that like this
```python
from graphql import GraphQLError
```